src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(a) {
+    if (a !== undefined) {
+      sum += a;
+
+      return adder;
+    } else {
+      const result = sum;
```

Comment:
Storing the result in a temporary variable is unnecessary. You could return `sum` directly before resetting it.